ophyd/_pyepics_shim.py

```diff
@@ -168,29 +137,32 @@ def get_pv(pvname, form='time', connect=False, context=None, timeout=5.0,
     if context is None:
         context = ca.current_context()
 
-    thispv = None
-
-    # TODO: this needs some work.
-    # thispv = epics.pv._PVcache_.get((pvname, form, context))
-    # if thispv is not None:
-    #     if callback is not None:
-    #         # wrapping is taken care of by `add_callback`
-    #         thispv.add_callback(callback)
-    #     if access_callback is not None:
-    #         access_callback = wrap_callback(_dispatcher, 'metadata',
-    #                                         access_callback)
-    #         thispv.access_callbacks.append(access_callback)
-    #     if connection_callback is not None:
-    #         connection_callback = wrap_callback(_dispatcher, 'metadata',
-    #                                             connection_callback)
-    #         thispv.connection_callbacks.append(connection_callback)
-    #     if thispv.connected:
-    #         if connection_callback:
-    #             thispv.force_connect()
-    #         if access_callback:
-    #             thispv.force_read_access_rights()
-
-    if thispv is None:
+    thispv = epics.pv._PVcache_.get((pvname, form, context))
```

Comment:
Any chance of pyepics giving us public API for this?

Reply:
You mean access to the `_PVcache_`?

Reply:
Replacing the majority of `get_pv` is now possible with pyepics directly, as of https://github.com/pyepics/pyepics/pull/158 (3.4.0rc2)

Reply:
Sorry for the slow response --- yes. Sounds good.